Taskfile.yml

```diff
@@ -190,13 +190,15 @@ tasks:
       END_BLOCK: '{{.END_BLOCK}}'
       LABELS: '{{.LABELS | default ""}}'
       BENCHMARK_OUTPUT_FILE: '{{.BENCHMARK_OUTPUT_FILE | default ""}}'
+      METRICS_DISABLED: '{{.METRICS_DISABLED | default ""}}'
```

Comment:
This is a bit of a non-standard flag - could we reverse it to `METRICS_ENABLED` and default to true